docs/source/posts/2021/2021-08-23-gsoc-devmessias-final-report.rst

```diff
@@ -0,0 +1,406 @@
+Google Summer of Code 2021 Final Work Product
```

Comment:
Why it appears a second time below?

Reply:
Ok,  thanks for pointing this out. I'll take look.

Reply:
could you add the header in the final report too?